dftimewolf/lib/collectors/grr_hosts.py

```diff
@@ -406,6 +413,9 @@ def SetUp(self,
     self.hostnames = [item.strip() for item in hosts.strip().split(',')]
     self.use_tsk = use_tsk
 
+    if action.lower() in self._ACTIONS:
+      self.action = self._ACTIONS[action.lower()]
+
```

Comment:
Would probably be better to bail if the action specified is not in `self._ACTIONS`

```python
self.action = self._ACTIONS.get(action.lower())
if not self.action:
  # add critical error
```

Reply:
Done. Had to use is None because stat enum is 0.